chain/chain/src/flat_storage_resharder.rs

```diff
@@ -1087,23 +1116,27 @@ mod tests {
             flat_storage_resharder.split_shard_task(parent_shard, split_params.clone());
         assert_matches!(task_result, FlatStorageReshardingTaskResult::Cancelled);
 
-        // Verify parent status is back to ready and children are cleaned up
-        let store = flat_storage_resharder.runtime.store().flat_store();
+        // Check that the resharding task was effectively cancelled.
+        // Note that resharding as a whole is not cancelled: it should resume if the node restarts.
```

Comment:
could we clarify here whether it means the operator needs to manually run the resume operation or that it would automatically resume?

Reply:
Yes sure, the node operation in this case would need to manually run the resume operation. Should I add a comment here?

Reply:
yeah maybe we can clarify the existing comment like:
it should resume if the node restarts. -> the node operator must run resume resharding before the node can restart